SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpAddressDto.cs

```diff
@@ -18,7 +18,7 @@ public class MpAddressDto
         [JsonProperty(PropertyName = "City")]
         public string City { get; set; }
 
-        [JsonProperty(PropertyName = "State")]
+        [JsonProperty(PropertyName = "State/Region")]
```

Comment:
Jim mentioned that - but we looked at crds-angular and it was unescaped. So you think we should still put brackets around it? Your call as Im not as experienced